api/src/main/java/org/pmiops/workbench/cohortreview/util/ParticipantConditionDbInfo.java

```diff
@@ -0,0 +1,38 @@
+package org.pmiops.workbench.cohortreview.util;
```

Comment:
Can we move stuff like this to like "workbench.cdm" or something so it can be reused?